src/schema/schema.graphql

```diff
@@ -1320,12 +1321,38 @@ type SignatureResponse {
 	signature: [NameId]
 }
 
+type Document {
+	id: ID
+	folderId: ID
+	name: String
+	version: Float #same item may have different versions (i.e same names) will need to implement ListDocumentRevisionsRequest 
+	contentType: String
+ 	descEnabled: Boolean
+	date: Float
+	changeDate: Float
+	modifiedSequence: Float
+	revision: Float
+	size: Float
+	sortField: String
+	tags: String # t
+	tagNames: String # tn
+	uuid: ID
+	folderUuid: String
+	metadataVersion: Float
+	leb: String
```

Comment:
use full names instead of shorter versions
leb: 'lastEditedAccount',
	cr: 'revisonCreator',
	cd: 'revisedCreationDate',
	loid: 'lockOwnerId',

Reply:
Done